packages/language-server/package.json

```diff
@@ -43,7 +43,8 @@
     "node-fetch": "2.6.1",
     "vscode-languageserver": "6.1.1",
     "vscode-languageserver-textdocument": "1.0.1",
-    "vscode-uri": "2.1.2"
+    "vscode-uri": "2.1.2",
+    "yaml": "^1.10.2"
```

Comment:
used fixed version?

Reply:
version 2.x is out

Reply:
I know, but this doesn't work with Node 12 anymore and you need to disable your Circle CI test, ok?

Reply:
Using fix version now `1.10.2` which is the last version supporting Node 12